tests/library/firefox/launcher.spec.ts

```diff
@@ -43,3 +45,34 @@ it('should pass firefox user preferences in persistent', async ({ mode, launchPe
   const error = await page.goto('https://example.com').catch(e => e);
   expect(error.message).toContain('NS_ERROR_PROXY_CONNECTION_REFUSED');
 });
+
+it('should support custom firefox policies', async ({ browserType, mode, asset, loopback }, testInfo) => {
+  it.skip(mode.startsWith('service'));
+
+  const policies = {
+    'policies': {
+      'Certificates': {
+        'Install': [asset('client-certificates/server/server_cert.pem')],
+      },
+    },
+  };
+  const policiesPath = testInfo.outputPath('policies.json');
+  await fs.promises.writeFile(policiesPath, JSON.stringify(policies));
+
+  const port = 48112;
```

Comment:
interesting port!

Reply:
Generated with a crypto-secure random generator in my head!